app/models/collection_number.rb

```diff
@@ -102,25 +107,68 @@ def log_destroy
   end
 
   def destroy_without_callbacks
```

Comment:
Actually, there might be a *much* better way to do this.  It's possible collection_number.delete does this for us!

Reply:
Hahaha - I'm not an Arel ninja, i'm more like a jarhead. I have a hammer and everything seems like a nail...! Now that i'm realizing that, there will probably be a bunch of cases where my Arel could be simplified with AR, and i've started on a few. 

The good thing about the Arel of course is that it's pretty explicit, less magical, so it's more evident if it's building something moronically overcomplicated. (Hopefully!)